src/services/api.js

```diff
@@ -126,6 +126,39 @@ const api = {
       }
     });
     return filteredComments;
+  },
+  async updateComment(commentUuid, updatedComment, entryUuid) {
+    const snapshot = await firebase
+      .firestore()
+      .collection('entries')
+      .get();
+
+    let commentsArray = [];
+    snapshot.docs.map(x => {
```

Comment:
What is the use of these lines? You are doing a `map`, but you don't use the result. Also, you are not doing any mutations in the `map` (which is good). But, leads me to believe that the code is useless.